### PR TITLE
loadkeys: add examples

### LINKED ISSUE #9003: page request: loadkeys
### Command description

Loads keyboard translation tables - https://manned.org/loadkeys.1

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by DEBADRIBASAK**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **2.3.0**
Closes #9003

### CODE REVIEW

File: pages/linux/loadkeys.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# loadkeys
+
+> Load keyboard translation tables.
+> More information: <https://manned.org/loadkeys.1>.
```
Comment (kbdharun) [inline pages/linux/loadkeys.md:4]:
```suggestion
> More information: <https://manned.org/loadkeys>.
```

File: pages/linux/loadkeys.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Load the default keymap:
+
+`loadkeys --default`
+
+- Load default keymap when an unusual keymap is loaded and - sign cannot be found:
```
Comment (kbdharun) [inline pages/linux/loadkeys.md:10]:
```suggestion
- Load default keymap when an unusual keymap is loaded and `-` sign cannot be found:
```

File: pages/linux/loadkeys.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`loadkeys defmap`
+
+- Load a keymap from the specified file for the console:
+
+`loadkeys --console {{/path/to/console}} filename`
```
Comment (kbdharun) [inline pages/linux/loadkeys.md:16]:
```suggestion
`loadkeys --console {{/path/to/console}} {{filename}}`
```

File: pages/linux/loadkeys.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- In Debian systems (like Ubuntu), install the package console-data using `sudo apt-get install console-data`. Then use standard names for keymaps of different locales:
```
Comment (kbdharun) [inline pages/linux/loadkeys.md:18]: I am not sure about this command description. @navarroaxel What do you think?

File: pages/linux/loadkeys.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`loadkeys --console {{path/to/console}} uk`
```
Comment (kbdharun) [inline pages/linux/loadkeys.md:20]:
```suggestion
`loadkeys --console {{path/to/console}} {{uk}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/loadkeys.md (hunk 4 above)
Comment (navarroaxel) [reply pages/linux/loadkeys.md:18]: We don't indicate installation instructions or how to turn on with `systemctl`

Comment (DEBADRIBASAK) [reply pages/linux/loadkeys.md:18]: Okay. I will remove that detail.

**Comment by DEBADRIBASAK** [comment]
@kbdharun Addressed the suggestions.

File: pages/linux/loadkeys.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`loadkeys --default`
+
+- Load default keymap when an unusual keymap is loaded and `-` sign cannot be found:
+
+`loadkeys defmap`
+
+- Load a keymap from the specified file for the console:
+
+`loadkeys --console {{/path/to/console}} {{filename}}`
```
Comment (navarroaxel) [inline pages/linux/loadkeys.md:16]: is this `/dev/tty1`? we have `/dev/sdXN` as a valid placeholder, so:
```suggestion
`loadkeys --console {{/dev/ttyX}} {{path/to/file}}`
```

File: pages/linux/loadkeys.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+> Load keyboard translation tables.
+> More information: <https://manned.org/loadkeys>.
+
+- Load the default keymap:
+
+`loadkeys --default`
+
+- Load default keymap when an unusual keymap is loaded and `-` sign cannot be found:
+
+`loadkeys defmap`
```
Comment (navarroaxel) [inline pages/linux/loadkeys.md:12]: Why is this useful vs `--default`?

**Comment by navarroaxel** [comment]
oh we merged another PR about `loadkeys` too. Could you resolve the conflicts and add the `--console` and with a specific locale examples?

File: pages/linux/loadkeys.md (hunk 7 above)
Comment (DEBADRIBASAK) [reply pages/linux/loadkeys.md:12]: It is useful if `-` is not found.

File: pages/linux/loadkeys.md (hunk 6 above)
Comment (DEBADRIBASAK) [reply pages/linux/loadkeys.md:16]: Yeah it can be /dev/ttyX or just /dev/console

**Comment by DEBADRIBASAK** [comment]
> 

I think the PR that got merged was for `lddd`

**Comment by DEBADRIBASAK** [comment]
> oh we merged another PR about `loadkeys` too. Could you resolve the conflicts and add the `--console` and with a specific locale examples?

One locale example is there. And I added that /dev/ttyX example.

**Review by navarroaxel — COMMENTED** [review]
Remove help and version examples.

File: pages/linux/loadkeys.md [hunk 8]
```diff
@@ -30,3 +34,11 @@
 - Display version:
 
 `loadkeys --version`
+
+- Load a keymap from the specified file for the console:
+
+`loadkeys --console {{/dev/ttyX}} {{/path/to/filename}}`
```
Comment (navarroaxel) [inline pages/linux/loadkeys.md:40]:
```suggestion
`loadkeys --console {{/dev/ttyN}} {{/path/to/filename}}`
```

File: pages/linux/loadkeys.md [hunk 9]
```diff
@@ -23,10 +27,10 @@
 
 `loadkeys --quiet`
 
-- Display help:
+- Load a keymap from the specified file for the console:
 
-`loadkeys --help`
+`loadkeys --console {{/dev/ttyN}} {{/path/to/filename}}`
```
Comment (navarroaxel) [inline pages/linux/loadkeys.md:32]:
```suggestion
`loadkeys --console {{/dev/ttyN}} {{path/to/file}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**